### PR TITLE
Changelog entry for The FlowFuse Assistant plugin

### LINKED ISSUE #2286: Changelog entry for "The FlowFuse Assistant"

### PR DESCRIPTION
**Pull request by Steve-Mcl**
closes #2286

## Description

Changelog entry for The FlowFuse Assistant plugin

**NOTE:** Do NOT merge until cloud settings are applied. See https://github.com/FlowFuse/flowfuse/issues/4073 for status

## Related Issue(s)

parent: #2286
owner: https://github.com/FlowFuse/flowfuse/issues/4073

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [x] Documentation has been updated

### CODE REVIEW

File: src/changelog/2024/07/flowfuse-assistant.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+---
+
+We are excited to announce the first release of The FlowFuse Assistant, a new plugin for your Node-RED instances designed to enhance your development experience.
+
+In this initial release, The FlowFuse Assistant can help you create function nodes for those times when a no-code solution is either not feasible or not possible.
+Additionally, it works directly inside the Function Node editor right where you need it.
+
+This is just the beginning. The FlowFuse Assistant will continue to evolve, with future updates bringing even more powerful features and enhancements.
+
+Starting from today, FlowFuse users can try this out on their instances.
```
Comment (joepavitt) [inline src/changelog/2024/07/flowfuse-assistant.md:17]: Will this be available to existing instances, new instances only, or is there something users need to do to upgrade and get the feature?

Comment (knolleary) [inline src/changelog/2024/07/flowfuse-assistant.md:17]:
```suggestion
Starting from today, FlowFuse Assistant is available on FlowFuse Cloud. Users will need to update their instances to the latest stack to enable it.
```

Comment (Steve-Mcl) [reply src/changelog/2024/07/flowfuse-assistant.md]: Nick made a suggestion and has been committed.  Please re-review. (resolves comment)

**Review by joepavitt — APPROVED** [review]

File: src/changelog/2024/07/flowfuse-assistant.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+title: The FlowFuse Assistant
+description: FlowFuse now allows selecting and comparing snapshots to visualize differences, facilitating easier understanding of changes between snapshots.
+date: 2024-07-05 13:00:00.0
```
Comment (Steve-Mcl) [inline src/changelog/2024/07/flowfuse-assistant.md:4]:
```suggestion
date: 2024-07-04 13:00:00.0

```

File: src/changelog/2024/07/flowfuse-assistant.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title: The FlowFuse Assistant
+description: FlowFuse now allows selecting and comparing snapshots to visualize differences, facilitating easier understanding of changes between snapshots.
```
Comment (Steve-Mcl) [inline src/changelog/2024/07/flowfuse-assistant.md:3]: Bah. Just realised my copy paste error. 

Don't merge just yet please.

**Comment by Steve-Mcl** [comment]
@joepavitt I have updated the article description. One last quick review please.

**Review by joepavitt — APPROVED** [review]

**Pull request merged**